### PR TITLE
✨ Implement [MagicGrid] to NFT pages

### PR DESCRIPTION
**Pull request by AuroraHuang22**
[MagicGrid reference](https://github.com/e-oj/vue-magic-grid/blob/master/test/src/components/magic-grid/magic-grid.js)
[rinkeby](https://rinkeby.liker.land/like1x0u9vdv7tnjfae6clqr5zjwh8trh5fmcjpqmtj)

### CODE REVIEW

**Comment by nwingt** [comment]
Seems the order is not quite right
https://rinkeby.liker.land/like1qv66yzpgg9f8w46zj7gkuk9wd2nrpqmca3huxf

image

**Comment by AuroraHuang22** [comment]
> Seems the order is not quite right https://rinkeby.liker.land/like1qv66yzpgg9f8w46zj7gkuk9wd2nrpqmca3huxf

Because the current API is to prioritize WritingNFT and then non-Writing NFT

**Comment by AuroraHuang22** [comment]
> Seems the order is not quite right https://rinkeby.liker.land/like1qv66yzpgg9f8w46zj7gkuk9wd2nrpqmca3huxf

The current ordering is to prioritize Writing NFTs and then non-Writing NFTs.

File: src/pages/dashboard/index.vue [hunk 1]
```diff
@@ -157,13 +133,21 @@ export default {
...
         if (newAddress) {
-          this.fetchUserInfo();
-          await this.loadNFTListByAddress(this.getAddress);
+          this.switchWalletCount += 1;
+          if (this.switchWalletCount < 2) {
+            this.fetchUserInfo();
+            await this.loadNFTListByAddress(this.getAddress);
+          } else {
+            this.$router.go();
+          }
```
Comment (nwingt) [inline src/pages/dashboard/index.vue:150]: How about renaming `switchWalletCount` to `hasSwitchedWallet`?

```suggestion
          if (!this.hasSwitchedWallet) {
            this.hasSwitchedWallet = true;
            this.fetchUserInfo();
            await this.loadNFTListByAddress(this.getAddress);
          } else {
            // XXX: Refresh the page for Magic Grid to work properly
            this.$router.go();
          }
```

**Comment by nwingt** [comment]
@AuroraHuang22 Can you rebase the branch? It seems it has mixed some commits of another features

**Pull request merged**